Improve How you can help page (#1989)
ppy/osu-wiki: wiki/How_You_Can_Help!/en.md

# How you can help!

> osu! itself has a large, and diverse community from all walks of life, and anyone can help make osu! more awesome for you, and for everyone.

Reviewer: This paragraph could be longer for a better introduction to the article. Maybe say what the article does (and look at the current version to clarify why help from the community is greatly appreciated)

## Help out with [the wiki](https://github.com/ppy/osu-wiki)

The wiki is our official knowledge base about anything and everything about osu! The wiki is looking for more contributors to make the site up to date with fresh new knowledge, and to reach a wider audience via translations.

> Read more about how you can get started contributing to the wiki [here!](https://osu.ppy.sh/help/wiki/osu!_wiki_Contribution_Guide)

Reviewer: You can shorten this link to `/wiki/osu!_wiki_Contribution_Guide` (I think this is the preferred form for wiki links)

## Help keep osu! friendly and safe

It's not always sunshine in the osu! community, so we have a dedicated team of moderators to keep the place comfy and safe. However, the moderation team keeps track of all 25 channels (and more!) in the chat, and a countless entries on the forums.

Fortunately, it's still easy for you to help out with the community, here's some things you can do to help:

- Follow your favorite channels at the osu! in-game chat, or at the forums in the web.
- If you see something happening which you think feels inappropriate or disturbs you in any way, use the `!report` command at the in-game chat to notify moderators of what's happening.
    - Don't worry about not having your report not being read, the moderators will always read your reports.

> ## Help answer technical, beatmap, and gameplay support

Reviewer: Header sounds weird, forgot a word here?
maybe generalise this to something like "Help by supporting other community members"

- If you see something happening that you think is inappropriate or disturbing in any way, use the `!report` command in the in-game chat to notify the moderators of it.

## Help other community members

> Lend someone a hand when they encounter an issue with the game! Whether its a driver issue, a beatmap question, or gameplay questions, if you can answer it, you help the person in solving their problem.

Reviewer 2: ```suggestion
Lend someone a hand when they encounter an issue with the game! Whether it is a driver issue, a beatmap question, or gameplay questions, if you can answer it, you help the person in solving their problem.
```

You can begin with lending people a hand on these following mediums:

- Choose a forum that sparks your interest:
   - [Technical Support](/community/forums/5)
   - [Beatmapping Support](/community/forums/56)
   - [Storyboarding](/community/forums/20)
   - [Gamplay and Rankings](/community/forums/13)

- Mark the forum as read using the link at the top right.
- Follow any new threads, reading each one and helping out when you can. You can use the unread icons to make sure you keep up.
- Read the [available guides](/wiki/List_of_Guides "List of Guides") to polish up your osu! knowledge. If the guides in there doesn't satisfy you, you can contribute to them!

## Help check/mod beatmaps!

> osu! has a dedicated team of reviewers called "modders" to check if the beatmaps are ready to be ranked and played by everyone. However, that doesn't stop you from contributing in this sector, so here's a few things you can do to help:

Reviewer: I wouldn't really call modders a team.. I think what the original meant here are the official teams (BNs, QAT)

- Look through or search the [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?m=-1&s=4) listing for any artists or songs you like (or just choose at random).
- Play the beatmaps and leave feedback in threads.
- Read the [Modding Guide](/wiki/Modding "Modding") and the [Ranking Criteria](/wiki/Ranking_Criteria "Ranking Criteria") to know the basics of technical modding and what makes a beatmap rankable.
- Join the `#modhelp` and `#modreqs` on in-game chat and join in the conversation (or just follow along).
- If you have the time and dedication, make a [Modding Queue](/community/forums/60) and have people start coming to you with requests.

> ## Join some user-run projects and contribute to making them happen (or try to make them better if they're active)

Reviewer: This could be shortened imho
something like "Help user-run [community-run?] projects related to osu!", maybe

> osu! has a thriving community of technical and competitive people. They range from tinkerers, to seasoned rhythm game veterans. There's a lot of proposals and projects being made exclusively for osu! and you can help them, or drive the future of osu! with your own creation!

Reviewer: i think the word "tinkerer" doesn't exist

Here's a few things you can do to help:

- Browse the [torunaments](/community/forums/55) forum. Tournaments takes time and a lot of manpower to organize without hitches, and you can help out by being a spectator judge.

> - Check out what's going on in [General Development](/community/forums/2). This low activity forum is where the [most](/community/forums/topics/58855) [serious](/community/forums/topics/58855) [projects](/community/forums/topics/80027) happen.

Reviewer: you repeat the same project here ("most" and "serious" link to the same project)

Also, the projects linked here are either dead, inactive, or very, very old (7yrs+). Maybe it'd be worth the minute to look for some newer, better examples

- Join a [beatmap project](/community/forums/53) and learn how fun collaborating with others can be!
- Start your own project! Whether its just bringing back dancing pippi, or just making a cool thing for osu!, anything is accepted.

> - Bring [osu!monthly](/community/forums/topics/75017) to life.

Reviewer: From what I can see, the original osu!monthly has been dead for a long time. Even the reboot (http://osumonthly.com/) hasn't released for a year now, so you can probably remove this point.

If you want to include this anyways, you should probably say "Bring ... **back** to life"

## Contribute to the future of osu!

Started last 2017, the Lazer project is the current focus of the osu! team, along with the new website, which you see today.

You can help making them even more better, and make us closer to the future of osu! by helping on these areas:

> - Helping with the [Lazer](https://github.com/ppy/osu) project. There is much to implement on the new version of osu!, and you can help make them happen!

Reviewer: I think the project name is "osu!lazer", but not sure about this one

Anyone can be a part of the community in their own way. Here are a few examples to start with:

## Help out with the wiki

[The wiki](https://github.com/ppy/osu-wiki) is our official knowledge base about anything and everything about osu!. The wiki is always looking for more contributors to help keep the site up to date with fresh new knowledge, and to reach a wider audience via translations.

For newcomers, you can read more about it in the [contributing guide](/wiki/osu!_wiki_Contribution_Guide).

## Help keep osu! friendly and safe

To keep the official community hubs safe, osu! has a dedicated moderation team that monitors them around the clock. However, they keep track of all 25 channels (and more!) in the chat, and countless entries in the forums.

Fortunately, it is still easy for you to help out keeping everything safe, here are some things you can do to help:

- Follow your favourite channels in the [osu! in-game chat](/wiki/Chat_Console), or in the forums on the web.
- If you see something happening that you think is inappropriate or disturbing in any way, use the `!report` command in the in-game chat to notify the moderators of it.

>     - Do not worry about having your report being left unread, the moderators always read reports.

Reviewer 3: i don't think this line is necessary.

osu! has a thriving community of technical and competitive people. They range from hobbyist tinkers, to seasoned rhythm game veterans. There are a lot of proposals and projects being made exclusively for osu! and you can help them, or drive the future of osu! with your own!

Here are a few things you can do to participate:

- Browse the [tournaments](/community/forums/55) forum. Tournaments take time and a lot of manpower to organize without hitches, and you can help out by being a spectator judge in some cases.
- Check out what is going on in [General Development](/community/forums/2). This low activity forum is where the most serious projects happen.
- Join a [beatmap project](/community/forums/53) and learn how fun collaborating with others can be!
- Start your own project! Whether it is bringing back dancing pippi, or just making something cool for osu!, anything is accepted.

## Contribute to the future of osu!

Started in 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which you see today.

You can help make them even better, and make us closer to the future of osu! by helping on these areas:

> - Helping with the [osu!lazer](https://github.com/ppy/osu) project. There is much to implement on the new version of osu!, and you can help make them happen!

Reviewer 3: ```suggestion
- Helping with the [osu!lazer](https://github.com/ppy/osu) project. There is much to implement on the new version of osu!, and you can help make it happen!
```

## Help out with [the wiki](https://github.com/ppy/osu-wiki)

The wiki is our official knowledge base about anything and everything about osu! The wiki is looking for more contributors to make the site up to date with fresh new knowledge, and to reach a wider audience via translations.

Read more about how you can get started contributing to the wiki [here!](/wiki/osu!_wiki_Contribution_Guide)

## Help keep osu! friendly and safe

To keep the official community hubs safe, osu! has a dedicated moderation team that monitors them around the clock. However, they keep track of all 25 channels (and more!) in the chat, and a countless entries on the forums.

Fortunately, it's still easy for you to help out keeping everything safe, here's some things you can do to help:

- Follow your favorite channels at the osu! in-game chat, or at the forums in the web.
- If you see something happening which you think feels inappropriate or disturbs you in any way, use the `!report` command at the in-game chat to notify moderators of what's happening.
    - Don't worry about not having your report not being read, the moderators will always read your reports.

## Help other community members

> Lend someone a hand when they encounter an issue with the game! Whether its a driver issue, a beatmap question, or gameplay questions, you can answer it if you spend enough time with osu!

Author: Keep in mind this was a paraphrase from the original article, and might need substantial modification to redo it which I will be doing happily

You can begin with lending people a hand on these following mediums:

- Choose a forum that sparks your interest:
   - [Technical Support](/community/forums/5)
   - [Beatmapping Support](/community/forums/56)
   - [Storyboarding](/community/forums/20)
   - [Gamplay and Rankings](/community/forums/13)

- Mark the forum as read using the link at the top right.
- Follow any new threads, reading each one and helping out when you can. You can use the unread icons to make sure you keep up.
- Read the [available guides](/wiki/List_of_Guides "List of Guides") to polish up your osu! knowledge. If the guides in there doesn't satisfy you, you can contribute to them!

## Help check/mod beatmaps!

osu! has a dedicated team of reviewers  to check if the beatmaps are ready to be ranked and played by everyone. However, that doesn't stop you from contributing in this sector, so here's a few things you can do to help:

- Look through or search the [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?m=-1&s=4) listing for any artists or songs you like (or just choose at random).

> - Play the beatmaps and leave feedback in threads.

Reviewer 4: ```suggestion
- Play the beatmaps and leave feedback in the beatmap discussion.
```

or if `threads`, probably `beatmap threads`?

- Read the [Modding Guide](/wiki/Modding "Modding") and the [Ranking Criteria](/wiki/Ranking_Criteria "Ranking Criteria") to know the basics of technical modding and what makes a beatmap rankable.

> - Join the `#modhelp` and `#modreqs` on in-game chat and join in the conversation (or just follow along).

Reviewer 4: ```suggestion
- Join the `#modhelp` and `#modreqs` on the in-game chat and join in on the conversation (or just follow along).
```

> - If you have the time and dedication, make a [Modding Queue](/community/forums/60) and have people start coming to you with requests.

Reviewer 4: ```suggestion
- If you have the time and dedication, make a [Modding Queue](/community/forums/60) to give beatmap creators a place to request you for mods.
```

> ## Help/join user-run community projects related to osu!

Reviewer 4: ```suggestion
## Help/join user-run community projects
```
I think you can omit the ` related to osu!` part in the heading?

> osu! has a thriving community of technical and competitive people. They range from hobbyist tinkers, to seasoned rhythm game veterans. There's a lot of proposals and projects being made exclusively for osu! and you can help them, or drive the future of osu! with your own creation!

Reviewer 4: ```suggestion
osu! has a thriving community of technical and competitive people. They range from hobbyist tinkers, to seasoned rhythm game veterans. There are a lot of proposals and projects being made exclusively for osu! and you can help them, or drive the future of osu! with your own!
```

> Here's a few things you can do to help:

Reviewer 4: ```suggestion
Here are a few things you can do to help:
```

> - Browse the [torunaments](/community/forums/55) forum. Tournaments takes time and a lot of manpower to organize without hitches, and you can help out by being a spectator judge.

Reviewer 4: ```suggestion
- Browse the [tournaments](/community/forums/55) forum. Tournaments take time and a lot of manpower to organize without hitches, and you can help out by being a spectator judge in some cases.
```

I think the `in some cases` part should be kept.

- Check out what's going on in [General Development](/community/forums/2). This low activity forum is where the most serious projects happen.
- Join a [beatmap project](/community/forums/53) and learn how fun collaborating with others can be!

> - Start your own project! Whether its just bringing back dancing pippi, or just making a cool thing for osu!, anything is accepted.

Reviewer 4: ```suggestion
- Start your own project! Whether it is bringing back dancing pippi, or just making something cool for osu!, anything is accepted.
```

Author: Grammar seems weird

## Contribute to the future of osu!

> Started last 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which you see today.

Reviewer 4: ```suggestion
Started in 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which you see today.
```
I think it should be `in`?

> You can help making them even more better, and make us closer to the future of osu! by helping on these areas:

Reviewer 4: ```suggestion
You can help make them even better, and make us closer to the future of osu! by helping on these areas:
```

- Helping with the [osu!lazer](https://github.com/ppy/osu) project. There is much to implement on the new version of osu!, and you can help make them happen!
- Make the [new website](https://github.com/ppy/osu-web) even better.

## Avoid being negative! Make people happy!

There's a old saying that you should become part of the solution, not the problem. You can help on osu! by making the community a brighter place. People complain, but there are many ways you can help ligthen up the mood in the community:

- Be friendly to each other. Life is too short to argue.

> - [Report cheaters when you see them](/wiki/Reporting_Bad_Behavior/Handing_Cheating "Handling Cheating"). Don't complain about them on other mediums, a simple `!report <username> seems to be cheating` can fix your problems within minutes or hours.

Reviewer 4: ```suggestion
- [Report cheaters when you see them](/wiki/Reporting_Bad_Behavior/Handing_Cheating "Handling Cheating"). Don't complain about them on other mediums, a simple `!report <username> seems to be cheating` can fix your problems within minutes to hours.
```

I think that `or` should be kept as `to`

- Go out of your way to mediate negative situations and turn them into positive outcomes.

> While this article cannot cover how much more you can help to osu!, but simply playing the game as well makes osu! even more awesome!

Reviewer 4: ```suggestion
While this article cannot cover how much more you can help the osu! community, simply playing the game as well can make osu! even more awesome!
```
